modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Apply.kt

```diff
@@ -315,4 +315,22 @@ interface Apply<F> : Functor<F> {
   ): Kind<F, Tuple10<A, B, C, D, E, FF, G, H, I, J>> =
     a.product(b).product(c).product(d).product(e).product(f).product(g)
       .product(h).product(i).product(j)
+
+  /**
+   * Given two actions, it performs them sequentially.
+   * Ignores the result of the first action.
+   *
+   * This is equivalent to *> in Haskell.
+   */
+  fun <A, B> Kind<F, A>.followedBy(fb: Kind<F, B>): Kind<F, B> =
+    map(this, fb) { (_, right) -> right }
+
+  /**
+   * Given two actions, it performs them sequentially.
+   * Discards the result of the second action.
+   *
+   * This is equivalent to <* in Haskell.
+   */
+  fun <A, B> Kind<F, A>.apIgnore(fb: Kind<F, B>): Kind<F, A> =
```

Comment:
we called this `flatTap`

Reply:
Oh, I saw the other discussion. I see!

Let's come up with a name for it.

Reply:
Well `flatTap` "taps" into the value but does not change it after a `flatMap` which is why it's actually a really good name imo.

This function applies an effect, but ignores the result, so I understand where `apIgnore` comes from, it just does not sound good :smile: 
`apEffect` might be something, `withEffect` also comes to mind. `alsoDo` sounds weird because of the "do", but maybe `apAlso` or `alsoAp` might be better.

Reply:
`apTap`

Reply:
I used to call flatTap, effectM. We agreed to use effect for other purposes, sadly.

Reply:
I'll use apTap so we don't stop this PR but following the reasoning above we are not tapping the value in the monad 🤔